src/index.ts

```diff
@@ -27,10 +27,13 @@ export interface Env {
 	DISCORD_PUBLIC_KEY: string;
 	DISCORD_BOT_TOKEN: string;
 	DISCORD_QUESTIONS_WEBHOOK: string;
-
+	DISCORD_QUESTION_CHANNEL: string;
```

Comment:
We can use the webhook to send the message here for consistency. Can use https://discord.com/developers/docs/resources/webhook#delete-webhook-message to delete the message when a new one is set